compiler/one-cmds/one-import-bcq

```diff
@@ -184,30 +184,41 @@ show_err_onexit()
 
 trap show_err_onexit ERR
 
-# generate temporary preserved pb file
-echo "${DRIVER_PATH}/preserve_bcq_info" --input_path ${INPUT_PATH} \
---output_path "${TMPDIR}/${MODEL_NAME}_preserved.pb"  > "${OUTPUT_PATH}.log"
+# Generate BCQ information metadata
+# If model has no BCQ information or invalid information, pb file is not changed.
+echo "${ROOT}/bin/generate_bcq_metadata" \
```

Comment:
Where came the `ROOT` ?

Reply:
Ooops.... I thought that I removed all of them :(
That was wrongly copied from `tf2nnpkg`... thanks T.T